### PR TITLE
NaturalNumbers as Iterable

### PR DESCRIPTION
**Pull request by Timmeey**
Step one towards funcAsIterator #101

### CODE REVIEW

**Comment by 0crat** [comment]
@timmeey I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/list/NaturalNumbers.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import java.util.Iterator;
+import java.util.stream.LongStream;
+
+/**
+ * Iterable providing Natural Numbers.
+ *
+ * @author Tim Hinkes (timmeey@timmeey.de)
+ * @version $Id$
+ * @since 0.4
```
Comment (yegor256) [inline src/main/java/org/cactoos/list/NaturalNumbers.java]: @Timmeey 0.6 is already released. The next one is 0.7.

**Comment by yegor256** [comment]
@Timmeey looks good, thanks. One small comment.

**Comment by Timmeey** [comment]
What?! How can it fail after changing the two version lines?!
@yegor256 ?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**